packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -304,7 +304,6 @@ export class OdspDocumentService implements IDocumentService {
                         && error.errorType === DriverErrorType.authorizationError) {
                         this.cache.sessionJoinCache.remove(this.joinSessionKey);
                     }
-                    assert(connection.disposed, "Connection should be disposed by now");
```

Comment:
Also we need to remove the assert in odspDocumentService because "disconnect" event is emitted https://github.com/microsoft/FluidFramework/blob/d38f46c16e20d14e88000fecb08433f72dcab2c2/packages/drivers/driver-base/src/documentDeltaConnection.ts#L481